README.md

```diff
@@ -0,0 +1,117 @@
+# Northcoders News API
+
+## Table of Contents
+
+- [About](#about)
+- [Getting Started](#getting-started)
+  - [Prerequsites](#prerequisites)
+  - [Cloning this repository](#cloning)
+  - [Installing dependencies](#dependencies)
+- [Database Management](#database-management)
+  - [Database setup](#database-setup)
+  - [Database seeding](#database-seeding)
+- [Special Thanks](#special-thanks)
+
+## About <a name = "about"></a>
+
+This API has been built for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as Reddit) which should provide this information to the front end architecture.
+
+This project is [hosted online here](https://wren-nc-news.onrender.com/api), so please take a look!
+
+The link will take you to a JSON file describing the endpoints accepted by the API, along with examples of accepted queries and typical responses.
+
+Please note: Your browser may require an extension to format the JSON file to be a little easier to read. For Chrome users, [this one](https://chromewebstore.google.com/detail/json-formatter/bcjindcccaagfpapjjmafapmmgkkhgoa?hl=en&pli=1) comes highly recommended!
+
+## Getting Started <a name = "getting-started"></a>
+
+These instructions will get a copy of this project running on your local machine for development and testing purposes!
+
+### Prerequisites <a name = "prerequisites"></a>
+
+We need to create a clone of the repository, install some dependencies and set up our testing and development databases.
+
+This API uses Node.js and PostgreSQL for database interactions, so please ensure your installation meets the following minimum requirements:
+
+```bash
+- Node v22.8.0
+- PostgreSQL v16.4
+```
+
+### Cloning this repository <a name = "cloning"></a>
+
+Open your terminal, navigate to the directory where you'd like to copy this repo, and run the following command:
+
+```bash
+git clone https://github.com/smlbrd/wren-nc-news.git
+```
+
+From here, you can create a new (empty!) repository on GitHub, link it to your clone and push it to your new repo:
+
+```bash
+git remote set-url origin <YOUR-REPO-URL>
+git push -u origin main
+```
+
+Now you should be all set to push any changes you make to your GitHub!
```

Comment:
This part doesnt need to be included as generally if someone would like to clone it down they probably wont want to make their own version by creating their own repo for it, so just the cloning instructions are fine

Reply:
Ah, perfect! I've updated the readme, thanks MKD!